.github/workflows/wheels.yml

```diff
@@ -70,6 +70,17 @@ jobs:
         - os: macos-latest
           python: "310"
           platform: macosx_x86_64
+          
+        # Windows builds
+        - os: windows-latest
```

Comment:
Maybe pin? Upgrading VS tends to change the DLLs required(at least from my experience with pandas), and there is no auditwheel/delocate on Windows.